src/test/kotlin/uk/gov/dluhc/registercheckerapi/rest/UpdatePendingRegisterCheckIntegrationTest.kt

```diff
@@ -877,8 +877,21 @@ internal class UpdatePendingRegisterCheckIntegrationTest : IntegrationTest() {
         )
     }
 
-    @Test
-    fun `should submit the POSTAL_VOTE result message to the postal vote queue`() {
+    @ParameterizedTest
+    @CsvSource(
+        value = [
+            "VOTER_CARD, VOTER_MINUS_CARD, 000000000000/confirm-applicant-register-check-result",
+            "POSTAL_VOTE, POSTAL_MINUS_VOTE, 000000000000/postal-vote-confirm-applicant-register-check-result",
+            "PROXY_VOTE, PROXY_MINUS_VOTE, 000000000000/proxy-vote-confirm-applicant-register-check-result",
+            "OVERSEAS_VOTE, OVERSEAS_MINUS_VOTE, 000000000000/overseas-vote-confirm-applicant-register-check-result",
+            "APPLICATIONS_API, APPLICATIONS_MINUS_API, 000000000000/register-check-result-response-queue-name",
+        ]
```

Comment:
Not entirely sure why the localStack config makes the URLs have these 0s but I have just opted to adding them here in the params rather than changing the configuration but happy to put abit more time into it if needed

Reply:
Is it possible to pass them in from localStackContainerSettings using string interpolation? Not sure whether that would work with how CsvSource works, but would be a bit neater. Bit worried about those 0s changing on different setups otherwise

Reply:
I don't think you can use string interpolation in a CsvSource or MethodSource as the values have to be static. Since the LocalStackConfig is AutoWired its not static so I don't think we can do that. I can keep investigating and maybe fix the weird 0s in the LocalStackConfiguration

Reply:
Good idea! I'll do that. Thanks